package.json

```diff
@@ -1,38 +1,26 @@
 {
   "name": "tmnt-client",
   "version": "0.0.1",
-  "description": "Provides a JavaScript client to TMNT's API.",
-  "main": "dist/node/client.js",
+  "description": "A JavaScript client library exposing TMNT's RESTful API.",
+  "main": "src/index.js",
   "repository": "git@github.com:ProjectPeter/tmnt-client.git",
   "author": "TMNT",
   "license": "MIT",
   "scripts": {
-    "start-node": "cross-env NODE_ENV=development nodemon --watch src --watch tests --exec babel-node tests/node.js",
+    "start-node": "cross-env NODE_ENV=development nodemon --watch src --watch tests --exec node tests/node.js",
     "start-browser": "cross-env NODE_ENV=development webpack-dev-server --progress --open --mode development --config webpack.config.js",
-    "build": "npm run build-browser && npm run build-node",
-    "build-node": "cross-env NODE_ENV=production npx babel src --out-dir dist/node/ --inline-source-maps",
-    "build-browser": "cross-env NODE_ENV=production webpack -p --progress --mode production --config webpack.config.js"
+    "build": "cross-env NODE_ENV=production webpack -p --progress --mode production --config webpack.config.js",
+    "watch": "cross-env NODE_ENV=development nodemon --watch src --watch tests --exec npm run build",
+    "publish-locally": "npm link",
```

Comment:
@vkuts this allows us to test it as an npm package locally before officially releasing

Reply:
That's def nice 👍